### ISSUE TITLE
get the dev-postgres branch to a minimal base state

### ISSUE THREAD

**Issue by xoldd** [PalisadoesFoundation/talawa-api#2596]
**Is your feature request related to a problem? Please describe.**
The `dev-postgres` branch is based off of talawa-api's `develop` branch(or the `main` branch I'm not sure) which is largely incompatible with the changes required for migration to postgres and thus cannot be incrementally migrated

The `dev-postgres` branch needs to be migrated to a minimal base state after which subsequent small changes can be made without compatibility problems. This branch will be manually synced with the `develop` branch. The plan is to get this branch merged with the `develop` branch before the amount of diversion in business logic becomes too big.  

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Approach to be followed (optional)**
A clear and concise description of approach to be followed.

**Additional context**

**Potential internship candidates**

Please read this if you are planning to apply for a Palisadoes Foundation internship 
- https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by palisadoes** [comment]
During the week of November 11, 2024 we will start a code freeze on the `develop` branches in Talawa, Talawa Admin and Talawa-API. 

We have completed a project to convert the Talawa-API backend to use PostgreSQL. Work will then begin with us merging code in the `develop` branches to a new `develop-postrgres` branch in each repository.

Planning activities for this will be managed in our #talawa-projects slack channel. A GitHub project will be created to track specially labeled issues. We completed a similar exercise last year using a similar methodology. 

Starting November 12, California time no new PRs will be accepted against the `develop` branch. They must be applied to the `develop-postrgres` branch. 

There are some GSoC project features that will need to be merged into develop. These will be the only exceptions.

This activity and the post GSoC 2024 start date was announced in our #general Slack channel last month as a pinned post.
